tools/iptb-plugins/filecoin/local/localfilecoin.go

```diff
@@ -327,13 +328,32 @@ func (l *Localfilecoin) RunCmd(ctx context.Context, stdin io.Reader, args ...str
 		return nil, err
 	}
 
-	stderrbytes, err := ioutil.ReadAll(stderr)
-	if err != nil {
-		return nil, err
-	}
+	g, ctx := errgroup.WithContext(ctx)
 
-	stdoutbytes, err := ioutil.ReadAll(stdout)
-	if err != nil {
+	var stderrbytes []byte
+	var stdoutbytes []byte
+
+	g.Go(func() error {
+		var err error
+		stderrbytes, err = ioutil.ReadAll(stderr)
+		if err != nil {
```

Comment:
we can shorten this to just `return err` I think, but i'm happy with it either way.

Reply:
Ya, it will be remove hopefully near the end of the week / next week so I'll probably just leave it so I can merge this PR.